taclib/cluster_role.yaml

```diff
@@ -18,15 +18,10 @@ metadata:
   name: taclib-role-binding
   namespace: default
 subjects:
-- kind: ServiceAccount
-  name: taclib-controller-sa
+- kind: Group
+  name: system:serviceaccounts
```

Comment:
This looks dangerous, what does it does this allow all the pods access to the K8s API?